src/ProjectTemplates/ProjectTemplatesNoDeps.slnf

```diff
@@ -1,14 +1,14 @@
 {
   "solution": {
-    "path": "ProjectTemplates.sln",
+    "path": "..\\..\\AspNetCore.sln",
     "projects": [
-      "ComponentsWebAssembly.ProjectTemplates\\Microsoft.AspNetCore.Components.WebAssembly.Templates.csproj",
-      "Web.Client.ItemTemplates\\Microsoft.DotNet.Web.Client.ItemTemplates.csproj",
-      "Web.ItemTemplates\\Microsoft.DotNet.Web.ItemTemplates.csproj",
-      "Web.ProjectTemplates\\Microsoft.DotNet.Web.ProjectTemplates.csproj",
-      "Web.Spa.ProjectTemplates\\Microsoft.DotNet.Web.Spa.ProjectTemplates.csproj",
-      "test\\ProjectTemplates.Tests.csproj",
-      "BlazorTemplates.Tests\\BlazorTemplates.Tests.csproj"
+      "src\\ProjectTemplates\\ComponentsWebAssembly.ProjectTemplates\\Microsoft.AspNetCore.Components.WebAssembly.Templates.csproj",
+      "src\\ProjectTemplates\\Web.Client.ItemTemplates\\Microsoft.DotNet.Web.Client.ItemTemplates.csproj",
+      "src\\ProjectTemplates\\Web.ItemTemplates\\Microsoft.DotNet.Web.ItemTemplates.csproj",
+      "src\\ProjectTemplates\\Web.ProjectTemplates\\Microsoft.DotNet.Web.ProjectTemplates.csproj",
+      "src\\ProjectTemplates\\Web.Spa.ProjectTemplates\\Microsoft.DotNet.Web.Spa.ProjectTemplates.csproj",
+      "src\\ProjectTemplates\\test\\ProjectTemplates.Tests.csproj",
+      "src\\ProjectTemplates\\BlazorTemplates.Tests\\BlazorTemplates.Tests.csproj"
```

Comment:
You'll need to keep the `ComponentsWebAssembly-CSharp.sln` file that follows. It's part of the project template

Reply:
Thanks, should be restored now.